samples/cellular/sms/src/main.c

```diff
@@ -78,6 +80,25 @@ int main(void)
 		return 0;
 	}
 
+	/* Setting SMS service center number if set into the configuration,
+	 * or if it's left empty, an information text is printed.
+	 */
+	if (strcmp(CONFIG_SMS_SERVICE_CENTER_NUMBER, "")) {
```

Comment:
It's easy to read this condition so, that the code below is executed when the configured service center number is empty, when it's actually the other way around. `sizeof() > 1` would be clearer, I think. Not sure if using `strcmp()` even generates code, which is unnecessary.

Reply:
Good point. Changed here and for `CONFIG_SMS_SEND_PHONE_NUMBER` below.